src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Data/Microsoft.Azure.Search.Data.csproj

```diff
@@ -23,7 +23,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Spatial" Version="7.5.3" />
-    <PackageReference Include="Microsoft.Azure.Search.Common" Version="[9.0.0, 10.0.0)" />
+    <PackageReference Include="Microsoft.Azure.Search.Common" Version="[9.0.1, 10.0.0)" />
```

Comment:
Would be a good idea to update `PackageReleaseNotes`

Reply:
Sure. @jadrefke will be sending a PR soon. Austin, can you update the release notes in `Microsoft.Azure.Search.csproj` as part of your change?

Reply:
Changed my mind; let's decouple these two PRs. I've updated the release notes in this PR.